docs/styles/demo/nav-docs.less

```diff
@@ -0,0 +1,5 @@
+.nav-demo {
+  .hxNav {
+    width: 15rem;
```

Comment:
`.hxNav` applies visual appearance, but `#nav` handles the width, so we imitate the width in the demo to maintain appearances in the layout.